psl-python/tests/data/models/simpleacquaintances.py

```diff
@@ -12,7 +12,7 @@
 
 DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'simple-acquaintances', 'data'))
 
-def run():
+def run(data_dir=DATA_DIR):
```

Comment:
Put spaces around equals, even for default arguments.
(This is in violation of PEP, but in-line with the rest of the code base.)